### PR TITLE
[doc] Use pangu plug-in to format Chinese documents

### PR DESCRIPTION
**Pull request by Miss-you**
### What this PR does / why we need it:

Use pangu plug-in to format Chinese documents

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

File: FAQ_CN.md [hunk 1]
```diff
@@ -197,15 +195,15 @@ Server: APISIX web server
 
 ## 如何修改日志等级
 
-默认的APISIX日志等级为`warn`，如果需要查看`core.log.info`的打印结果需要将日志等级调整为`info`。
+默认的 APISIX 日志等级为`warn`，如果需要查看`core.log.info`的打印结果需要将日志等级调整为`info`。
```
Comment (juzhiyuan) [inline FAQ_CN.md:198]: Do we need a space before `warn`?

File: FAQ_CN.md [hunk 2]
```diff
@@ -197,15 +195,15 @@ Server: APISIX web server
...
 
 具体步骤：
 
-1、修改conf/config.yaml中的nginx log配置参数`error_log_level: "warn"`为`error_log_level: "info"`。
+1、修改 conf/config.yaml 中的 nginx log 配置参数`error_log_level: "warn"`为`error_log_level: "info"`。
```
Comment (juzhiyuan) [inline FAQ_CN.md:202]: use `conf/config.yaml` please

File: FAQ_CN.md [hunk 3]
```diff
@@ -220,3 +218,25 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
 ```shell
 apisix reload
 ```
+
+## 如何格式化中文文档？
+
+中文文档对于格式有很多要求，比如英文单词前后要有空格` `，但每次手动修改都非常繁琐，有没有省时省力的办法解决这个问题？
```
Comment (juzhiyuan) [inline FAQ_CN.md:224]: We'd better use a more direct way to tell users what happened and how to deal with it.

e.g: 中文文档对于格式有很多要求：如英文单词前后应有空格` `，由于手动修改十分繁琐，我们在 VS Code 中使用 [`Pangu-Markdown`](https://marketplace.visualstudio.com/items?itemName=xlthu.Pangu-Markdown)插件解决该问题。

File: FAQ_CN.md [hunk 4]
```diff
@@ -220,3 +218,25 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
...
+
+**安装方式**
+
+进入 vscode 插件列表，搜索`pangu`，安装`Pangu-Markdown`插件
+
+![pangu](doc/images/pangu-plugin-1.png)
+
+**使用方式**
+
+进入 vscode 命令行模式，选择 `Pangu Format`
```
Comment (juzhiyuan) [inline FAQ_CN.md:236]: use `Command + P` to show the commands panel on Mac?

File: FAQ_CN.md [hunk 5]
```diff
@@ -220,3 +218,25 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
...
+
+![pangu](doc/images/pangu-plugin-2.png)
+
+**注意**
+
+不建议选用 pangu 这个名字的插件，它对 `**` 这种加粗字符支持不友好
```
Comment (juzhiyuan) [inline FAQ_CN.md:242]: We could recommend users to use the correct `Pangu-Markdown` plugin directly.

File: FAQ_CN.md (hunk 4 above)
Comment (Miss-you) [reply FAQ_CN.md:236]: shift + command + p

**Comment by membphis** [comment]
@Miss-you need to rebase your branch with `master`.

**Pull request closed without merging**